lib/environments/meteorEnvironment.js

```diff
@@ -321,13 +321,7 @@ export default {
     ['.html', '.css'].forEach((ext: string) => {
       const moduleSpecifier = `${basePath}${ext}`;
       if (fs.existsSync(path.join(config.workingDirectory, moduleSpecifier))) {
-        if (config.get('useRelativePaths')) {
-          moduleSpecifiers.push('.'.concat(path.sep, path.basename(moduleSpecifier)));
-        } else {
-          // Strip the leading '.' off of the moduleSpecifier to turn it into a Meteor compliant
-          // absolute path.
-          moduleSpecifiers.push(moduleSpecifier.slice(1));
```

Comment:
What if we changed `moduleNameFormatter` into an array of `moduleNameTransformers`? 

```
moduleNameTransformers: [
  ({ moduleName }) => moduleName.replace(/\jsx?$/, ''), // what used to be stripFromPath
  ({ moduleName }) => moduleName.replace(/^app/, ''), // what used to be lookupPath
],

We could merge this with whatever is in the default configuration/environment. 

The only option I have a hard time seeing here is `useRelativePaths`, because that's something that we want by default, but allow people to turn off.